academy/0.0-B9lab-Blockchains/3_managed.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Managed Blockchains"
+order: 3
+description: Introduction to Managed Networks
+tag: fast-track
+---
+
+# Introduction to Managed Networks
+
+Managed networks, just like public networks, rely on blockchain data structures. But unlike public blockchain networks they do not *necessarily* need to mitigate the Byzantine General's Problem because they operate in a predictable environment with elements of authority, hierarchy and accountability.
+
```

Comment:
Completely missing here in my opinion is an introduction of what managed networks are, as you dive right into the comparison. What do we mean by private/managed?

Reply:
## Managed Blockchain Networks

Managed networks, just like public networks, rely on blockchain data structures. Unlike public blockchain networks, they do not necessarily need to mitigate the Byzantine Generals' Problem because they operate in a predictable environment with elements of authority, hierarchy, and accountability.

Managed networks can be used in cases where elements of trust already exist between the participants. For example, consider a network of financial institutions. They could use blockchain technology to settle inter-bank transactions. In that case, there would be no need for public access and, indeed, public access would be undesirable. Instead, they might consider a private network with each institution operating exactly one validator. All participants would understand that no operator in a single institution would be able to corrupt the network. In fact, it would be much easier for them, as a group, to alter a history if they found it detrimental since it would only require the coordinated action of a small number of known validators. 

Managed networks are typically governed through traditional governance processes that are appropriate for the shared goals of the participants.